Cabal/Cabal.cabal

```diff
@@ -1,5 +1,5 @@
 name:          Cabal
-version:       1.25.0.0
+version:       2.0.0.0
```

Comment:
I want this temporarily, to see if I can force custom setups of test dependencies to be build with system `Cabal`